src/StreamsAPI_Lambda/Students/Students.java

```diff
@@ -0,0 +1,83 @@
+package StreamsAPI_Lambda.Students;
+
+public class Students {
+        private String firstName;
+        private String lastName;
+        private int age ;
+        private String gender;
+        private String departamentName ;
+        private int averageGrade;
+
+    public Students(String firstName, String lastName, int age, String gender, String departamentName, int averageGrade) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
+        this.gender = gender;
+        this.departamentName = departamentName;
+        this.averageGrade = averageGrade;
+    }
+
+    public Students() {
```

Comment:
it looks like you don't need this constructor and setters - you set all in the constructor line 11. for the future: you can use [Lombok](https://projectlombok.org/) and its [@Data](https://projectlombok.org/features/Data) would be what you need here